CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
 The development guide is located under the [docs](docs/README.md) folder.
+
+# Contribution guidelines 
+
+Welcome to [oceanbase]! We're thrilled that you'd like to contribute. Your help is essential for making it better.
+
+## Getting Started
+
+Before you start contributing, please make sure you have read and understood our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Fork the Repository
+
+First, fork the [repository](https://github.com/oceanbase/oceanbase) to your own GitHub account. This will create a copy of the project under your account.
+
+
+### Clone the Repository
+```
+git clone https://github.com/oceanbase/oceanbase
+```
+### Navigate to the project directory 📁
+```
+cd oceanbase
+```
+Create a new branch for your feature or bug fix:
+```
+ git checkout -b feature-branch
+ ```
+
+Make your changes and commit them:
+```
+git add .
+git commit -m "Description of your changes"
+```
+Push your changes to your fork:
+```
+git push origin feature-branch
```

Comment:
I think most of users don't have permission pushing code to origin.

Reply:
is this command ok then ?
git push feature-branch

Reply:
No. Usually, we fork the origin repo to our repo and checkout it local, and then create a branch and then push it to remote, and then we create a pull request.